queue.go

```diff
@@ -62,3 +63,36 @@ type Topic string
 
 // TopicRouter is used to determine the destination topic for an model.APMEvent.
 type TopicRouter func(event model.APMEvent) Topic
+
+// NewEventTypeTopicRouter returns a topic router which routes events based on
+// an optional prefix, and the event.Processor.Event field for an event.
+func NewEventTypeTopicRouter(prefix string) TopicRouter {
```

Comment:
Does it make sense to keep this in the loadgen package or do you forsee this being used elsewhere too?

Reply:
I think we're likely to reuse it for other producers.